Dockerfile

```
@@ -95,6 +95,12 @@ USER root:root
 
 WORKDIR /app
 
+# Install psql and mysql
+RUN apt-get update && \
+    apt-get install -y postgresql-client && \
+    apt-get install -y default-mysql-client && \
```

you should be able to write it in one line ?
```suggestion
    apt-get install -y postgresql-client default-mysql-client && \
```